packages/time-select/src/time-select.vue

```diff
@@ -1,17 +1,21 @@
 <template>
   <el-select
-    v-model="value"
+    :model-value="value"
     :disabled="!editable"
     :clearable="clearable"
     :clear-icon="clearIcon"
     :size="size"
     :placeholder="placeholder"
     default-first-option
     filterable
+    @update:model-value="
+      (value) => {
+        $emit('update:modelValue', value)
+      }
+    "
     @change="
```

Comment:
I don't quite understand why do you have to do this
since `v-model` is a syntax sugar for two way bindings which will be unwraps to `modelValue` and `update:modelValue`.
What is the point of rewriting this?

Reply:
I think I might know the root cause on this issue, the simplest way to fix this utilizing the props.modelValue instead of wrapping it with `computed`.

Reply:
cc @zazzaz

Reply:
The easiest way to fix this is that we bind it with `props.modelValue` instead of creating a new value, could you try that locally see if it works?

Reply:
Nope, `v-model` is shorten for `:model-value="v"` and `@update:model-value="v = $event"` (not `update:model-value="$emit('update:model-value', $event)"`), which will cause a error when clicking an option item in dropdown menu. I'll add a test case for that later.
![image](https://user-images.githubusercontent.com/3152496/113118266-ddfff200-9241-11eb-9c65-8e185164e9c5.png)

Reply:
Also, maybe related to #1505 ?

Reply:
Right, make sense, I was wrong about the `@update` thing.  LGTM.